nss-day-cohort-48/giffygram-webcrawlers
# Bk read messages

#### Changes Made
1. Modified file `MessageList, Provider, and Navbar` to include `filtering read messages and ability to mark as read and delete a message` and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout bk-readMessages
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: When user does BLANK, then BLANK should happen.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
